sw/airborne/firmwares/rotorcraft/guidance/guidance_h.c

```diff
@@ -298,6 +298,11 @@ void guidance_h_run(bool  in_flight)
       if ((!(guidance_h.mode == GUIDANCE_H_MODE_FORWARD)) && transition_percentage > 0) {
         transition_run(false);
       }
+
+#ifdef COMMAND_THRUST_X
+      stabilization_cmd[COMMAND_THRUST_X] = radio_control.values[RADIO_CONTROL_THRUST_X];
+#endif
+
```

Comment:
I really don't like to have something so specific here. What about adding this line here :https://github.com/paparazzi/paparazzi/blob/master/sw/airborne/firmwares/rotorcraft/stabilization/stabilization_indi.c#L628, which corresponds to the case where stabilization indi is called without guidance if I'm not wrong ?

Reply:
OK, I moved it.